.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
I'd love to have it, but something in CI would need to enforce it (the linter typically does this for us). Do you know of anything that can do that?